go-controller/pkg/ovn/kubevirt.go

```diff
@@ -0,0 +1,60 @@
+package ovn
```

Comment:
Is desgined as methods with DefaultNetworkController as receiver similar to pods to have access to all the clients there and similar stuff there.

An alternative would be to have "free" functions at "kubevirt" package or create a exported struct with those free functions as methods and put whatever we need at the struct from DefaultNetworkController, what do you think ?

Reply:
I missed reviewing the routing commit so I was not aware of everything. 

Some stuff though looks like it could be moved over like `ensureNetworkInfoForVM`. Other also seems trivial to convert to free functions if they just take the additional database client.

But I forfeit this comment until I review the commit that I missed.

Reply:
I will try to move as much as possible